src/styles/blocks/card.css

```
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  border-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-dark-grey);
+
+  margin-bottom: 4px;
+}
+
+.card__code-number {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-light-grey);
+  margin-bottom: 16px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__price-block {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__price-label {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-light-grey);
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--color-dark-grey);
+}
+
+.card__buy-link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 40px;
+
+  background: var(--color-blue);
+  border: 1px solid transparent;
+  border-radius: 5px;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+
+  text-transform: uppercase;
+  color: #fff;
+
+  transition: 0.3s;
```

It would be best if you explicitly named all properties that will change on transition
```suggestion
  transition: 0.3s property1 ease, 0.3s property2 ease;
```